content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,172 @@
+
+### ✍️ Free Draw mode
+
+The [free hand draw](https://github.com/excalidraw/excalidraw/pull/1570) was one of the most [requested](https://github.com/excalidraw/excalidraw/issues/25) features. Thanks to [kbariotis](https://github.com/kbariotis) for implementing this.
+
+https://twitter.com/excalidraw/status/1260287781596794880
+
+### #️⃣ Grid and Stats
+
+From early on people were asking for more precision in their hand drawn diagrams. We complied by adding the [grid support](https://github.com/excalidraw/excalidraw/pull/1788) implemented by [dai-chi](https://github.com/dai-shi) and—inspired by _YouTube's stats for nerds_—our own version of [stats](https://github.com/excalidraw/excalidraw/pull/2453) implemented by [lipis](https://github.com/lipis). Both features could be found under the context menu from the canvas.
+
+### 💾 File system integration and file handling
+
+In Excalidraw, we use the [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) library to integrate with the file system of the operating system. This allows us to support a true open→edit→save workflow with proper over-saving and save-as on supported browsers, with a fallback to file uploads and downloads on other browsers. Read more about this feature in [tomayac](https://github.com/tomayac)'s [earlier article](/browser-nativefs/) on this blog. We also have [experimental support](https://web.dev/file-handling/#demo) for file type association, so that when you double-click an `.excalidraw` file in your file explorer, the Excalidraw PWA opens.
+
+### ⚙️ Gatsby plugin
+
+We also have a plugin for Gatsby that automatically converts links to saved Excalidraw drawings to inline SVG at build time. We are actually using it in this post for our charts. Implemented by [trevorblades](https://github.com/trevorblades) & [j-f1](https://github.com/j-f1), and you can find it under [@excalidraw/gatsby-embedder-excalidraw](https://github.com/excalidraw/gatsby-embedder-excalidraw).
+
+https://twitter.com/Vjeux/status/1257906664239333376
+
+### 📦 npm package
+
+One of the last things we've introduced this year was a completely new npm package, available at [`@excalidraw/excalidraw`](https://www.npmjs.com/package/@excalidraw/excalidraw). A long time in the making (thanks to [@aakansha1216](https://twitter.com/aakansha1216) for most of the work), this package allows you to easily embed Excalidraw as a React component into your apps.
```

Comment:
this should probably be the last thing as it also says `one of the last things` :)

Reply:
do it

Reply:
I don't have access to push to this repo. Could you give me access @lipis?

Reply:
now you have

Reply:
Thank you 👍